README.md

```diff
@@ -1,6 +1,6 @@
 # Simple S3 Resource for [Concourse CI](http://concourse.ci)
 
-Resource to upload files to S3. Unlike the [the official S3 Resource](https://github.com/concourse/s3-resource), this Resource doesn't care about files being versioned.
```

Comment:
I should clarify: there's a notion of versioning at the S3 level, where a single file can have multiple versions, as opposed to a file naming convention for different versions (e.g. `mydownload-0.1.2.tar.gz`). https://github.com/concourse/s3-resource#file-names explains the difference.

I hadn't actually realized before today that a) S3 has the former capability, and b) that https://github.com/concourse/s3-resource supports it. In other words, I think **we could have used that Resource all along**.

On a brighter note, my GIF search for "facepalm" was rather fruitful:

![facepalm 1](http://memesvault.com/wp-content/uploads/Facepalm-Meme-Gif-15.gif)

![facepalm 2](http://gif-finder.com/wp-content/uploads/2015/09/Alice-Facepalm.gif)

![facepalm 3](https://media.giphy.com/media/rXMkGj2Z3iKGs/giphy.gif)

![facepalm 4](http://meowgifs.com/wp-content/uploads/2014/12/Facepalm-cat1.gif)